TrafficPlugin/MBXTrafficPlugin.m

```diff
@@ -46,30 +46,29 @@ - (void) setupPropertiesFor:(MGLMapView *)mapView {
         NSBundle *bundle = [NSBundle bundleForClass:[MBXTrafficPlugin class]];
         _bundleIdentifier = [bundle bundleIdentifier];
         NSString *sourceIdentifier = [NSString stringWithFormat:@"%@-traffic-source", _bundleIdentifier];
-        _source = [[MGLVectorSource alloc] initWithIdentifier:sourceIdentifier configurationURL:[NSURL URLWithString:@"mapbox://mapbox.mapbox-traffic-v1"]];
+        _source = [[MGLVectorTileSource alloc] initWithIdentifier:sourceIdentifier configurationURL:[NSURL URLWithString:@"https://raah.ir/dynamic/traffic.json"]];
```

Comment:
In order for this PR to be merged, the configuration URL would have to be changed back to a mapbox:// URL under the Mapbox user account.